src/Avalonia.Native/AvaloniaNativePlatformExtensions.cs

```diff
@@ -68,6 +68,13 @@ public class MacOSPlatformOptions
         /// </summary>
         public bool ShowInDock { get; set; } = true;
 
+        /// <summary>
+        /// If you will distribute you app in Mac App Store - it should be with sandbox enabled.
+        /// This parameter enables <see cref="Application.SandboxBookmarkAdded"/> event.
+        /// It's called when a user selects a document in OpenFileDialog or SaveFileDialog or via <see cref="Application.UrlsOpened"/> event.
+        /// </summary>
+        public bool AppStoreSandbox { get; set; } = false;
```

Comment:
I am not really familiar with macos sandboxing, but do we need this property? Is it a problem having always enabled event for sandbox? I assume it just will never be called in non-sandboxed app.

Reply:
before calling this event I'm creating a sandbox bookmark for selected path from NSSavePanel/NSOpenPanel, I did not test it, but I think not calling sandbox apis in not sandboxed app is a good idea.